### DISCUSSION TITLE
show the most interesting play on frontage and have next button to navigate to rest

### DISCUSSION THREAD

**Discussion by Deepak8717** [reactplay/react-play#226] [Ideas]
As more and more plays are added its kind of easy to feel lost and overwhelmed for better user experience we can show the fewer/interesting plays on front screen and have next button at the bottom just like codepen does. 

We can also showcase by category on front page like few plays of beginner level followed separate grid of intermediate level and so on. 

We can also have option to collaborate with teammates on advance project where we can divide the work.

**Comment by atapas** [comment]
Yes, I am also thinking of a similar line. We have 20 plays now we already feel it crowded. I shall work with @nirmalkc to come up with a design this weekend.

**Comment by koustov** [comment]
We need to prepare the base data point to understand which is the one that should be featured.
